scipy-1.0/covlett.tex

```diff
@@ -56,8 +56,8 @@
 
 We are submitting the enclosed manuscript describing a milestone
 1.0 release of SciPy for consideration to be published in
-\emph{Scientific Reports}. SciPy is the fundamental scientific 
-computing library in the Python programming language. 
+\emph{Scientific Reports}. SciPy is the fundamental scientific
+computing library in the Python programming language.
```

Comment:
Probably don't need to clean up the cover letter since #193 will adjust it for a new journal anyway & it is not really part of any source submission, I don't think.